canonical-web-and-design/maas-ui
# Fix Loader in paragraph error

## Done
- Upgrade React Components.
- Replace Loader with Spinner.

## QA
- Visit /r/machines.
- Select some machines.
- Using the take action menu select the option to change the pool.
- Select a pool (or create a new one).
- A spinner should appear with the status of the machines.
- In your console there shouldn't be an error about a `div` not being allowed in a `p`.
